charters/PORTABILITY/charter.md

```diff
@@ -0,0 +1,57 @@
+# Pipeline Portability Service
+
+## Description
+
+The portability service enables the confirmation that pipelines developed for the HCA DCP can be successfully ran outside of the HCA DCP.
+
+## Objectives
+
+The portability service provides the ability to run HCA DCP pipelines 
+
+## In-scope
+
+* Programmatic authorized access to submit pipelines that will be ran to confirm they execute to completion on different computing infrastructure (execution environments)
+* Programmatic authorized access to extend the backend of the portability service to include additional execution environments
+* A UI to describe the current state of submitted pipelines and execution environments
+* Documentation to describe how to submit a backend or pipeline the system
+* Documentation on portability best practices
+
+### Scientific "guardrails"
+
+Portability service will not comment on scientific accuracy of pipelines, with this constraint it will not need guardrails
```

Comment:
The use of _comment_ struck me as odd for a service that's not Siri. How about:

- will not address the accuracy
- will not report on the accuracy
- will not assess the accuracy
- will not validate the accuracy

Reply:
Lol, ok fair enough. I changed to using some of your power words (assess and report), see in the upcoming push.